app/uk/gov/hmrc/vatapi/config/FeatureSwitch.scala

```diff
@@ -40,7 +40,7 @@ case class FeatureSwitch @Inject()(value: Option[Configuration], env: String) {
       case Some(config) =>
         config
           .getStringSeq("white-list.applicationIds")
-          .getOrElse(throw new RuntimeException(s"$env.feature-switch.white-list.applicationIds is not configured"))
+          .getOrElse(throw new RuntimeException(s"feature-switch.white-list.applicationIds is not configured"))
```

Comment:
? Inject RunMode to get an env for the key?

Reply:
As discussed yesterday, we will make changes to the app-config-xxx